### ISSUE TITLE
코드 속 test set 문제 

### ISSUE THREAD

**Issue by wheemyungshin** [DatathonInfo/MISOChallenge-musculoskeletal#11]
Load_data.ipynb 속
```

dcm_list = sorted(glob.glob('../test/*.dcm'))
json_list = sorted(glob.glob('../test/*.json'))

```
이나

Train.ipynb 속

```
test_image_array = np.load('../dataset/test_image.npy')
test_label_array = np.load('../dataset/test_label.npy').reshape(-1, 16)

....

X_test = test_image_array[:] / 255
y_test = test_label_array[:] / 512

....

# Train
history = model.fit(X_train, y_train, batch_size=20, epochs=1000, shuffle=True, verbose=1, 
                    validation_data=(X_test, y_test), callbacks=callbacks_list)
```
처럼 test set을 사용하는 코드가 있는데 test set은 데이터 안에 없습니다.
이 코드는 무시하고 test set 대신 validation set을 넣어 사용하면 될까요?

감사합니다

**Comment by DatathonInfo** [comment]
안녕하세요 참가자님,

Validation set을 사용해주시면 됩니다

감사합니다
